Library/brew.rb

```diff
@@ -98,7 +98,7 @@ def require?(path)
   cmd = HOMEBREW_INTERNAL_COMMAND_ALIASES.fetch(cmd, cmd)
 
   sudo_check = %w[ install reinstall postinstall link pin unpin
-                   update upgrade create migrate tap switch ]
+                   update upgrade create migrate tap switch fix-permissions ]
```

Comment:
Seems like maybe we should allow this and just default to `--force` if you're using `sudo`?